.github/scripts/check_arch_symbols.py

```
@@ -0,0 +1,104 @@
+import os, re, subprocess, sys, glob
+
+VARIANT = re.compile(
+    r"^_?((crc32|adler32)_(braid|chorba\w*|pclmulqdq|vpclmulqdq\w*|armv8\w*|"
+    r"power8|s390_vx|riscv64_zbc|loongarch64|c|sse2|ssse3|sse42|avx2|avx512\w*|"
+    r"neon|vmx|rvv|lsx|copy_\w+)|compare256_\w+|longest_match\w*|slide_hash_\w+|"
+    r"chunkmemset_safe_\w+|inflate_fast_\w+)$"
+)
+
+def nm(args):
+    try:
+        return subprocess.run(args, capture_output=True, text=True).stdout
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)